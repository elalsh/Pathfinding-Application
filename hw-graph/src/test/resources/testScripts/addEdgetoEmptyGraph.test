# Tests adding a an a single directed edge to the graph

# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 A
AddNode graph1 B

# Add an edge
AddEdge graph1 A B edgeAB

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 A