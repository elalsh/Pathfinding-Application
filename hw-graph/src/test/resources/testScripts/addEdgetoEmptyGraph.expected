# Tests adding a an a single directed edge to the graph

# Create a graph
created graph graph1

# Add a pair of nodes
added node A to graph1
added node B to graph1

# Add an edge
added edge edgeAB from A to B in graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: A B
the children of A in graph1 are: B(edgeAB)